libpod/container_internal_common.go

```diff
@@ -2882,8 +2882,12 @@ func (c *Container) fixVolumePermissions(v *ContainerNamedVolume) error {
 			return err
 		}
 
-		// Make sure the new volume matches the permissions of the target directory.
+		// Make sure the new volume matches the permissions of the target directory unless 'U' is provided.
```

Comment:
nit: It would make sense to point out here that U is already chowned before here in generateSpec() in case somebody wonders why.